### DISCUSSION TITLE
Add `git add -A`

### DISCUSSION THREAD

**Discussion by lab596** [hack4impact-calpoly/starter-pack#8] [Ideas]
Hey, in Lab 10 Topic 5 it may be worth adding that users should use the command `git add -A` to add all the files to the track at the same time if they decide to add all the files in one commit.

**Comment by khoa-l** [comment]
Good call! Although I would advise against `git add -A` as that adds all files in the entire repository. Use `git add .` (period) instead. It will only add files in the current directory and sub-directories.

**Reply by lab596** [reply]
Oh I see, thanks for the suggestion!
